Shubham_Sharma/Tic-Tac-Toe/Assets/js/Main.js

```diff
@@ -0,0 +1,167 @@
+var options = "X";
+
+//Resets every button's text and color
+function reset(){
+    var b1 = document.getElementById("b1");
+    var b2 = document.getElementById("b2");
+    var b3 = document.getElementById("b3");
+    var b4 = document.getElementById("b4");
+    var b5 = document.getElementById("b5");
+    var b6 = document.getElementById("b6");
+    var b7 = document.getElementById("b7");
+    var b8 = document.getElementById("b8");
+    var b9 = document.getElementById("b9");
+
+    b1.innerHTML = "";
+    b2.innerHTML = "";
+    b3.innerHTML = "";
+    b4.innerHTML = "";
+    b5.innerHTML = "";
+    b6.innerHTML = "";
+    b7.innerHTML = "";
+    b8.innerHTML = "";
+    b9.innerHTML = "";
+
+    b1.style.backgroundColor = "#a52a2a";
+    b2.style.backgroundColor = "#a52a2a";
+    b3.style.backgroundColor = "#a52a2a";
+    b4.style.backgroundColor = "#a52a2a";
+    b5.style.backgroundColor = "#a52a2a";
+    b6.style.backgroundColor = "#a52a2a";
+    b7.style.backgroundColor = "#a52a2a";
+    b8.style.backgroundColor = "#a52a2a";
+    b9.style.backgroundColor = "#a52a2a";
+
+    b1.disabled = false;
+    b2.disabled = false;
+    b3.disabled = false;
+    b4.disabled = false;
+    b5.disabled = false;
+    b6.disabled = false;
+    b7.disabled = false;
+    b8.disabled = false;
+    b9.disabled = false;
+}
+
+//Checks for any WINNING pattern, and will call a method for highlighting those button's color
+function checker(){
+    var b1 = document.getElementById("b1").innerHTML;
+    var b2 = document.getElementById("b2").innerHTML;
+    var b3 = document.getElementById("b3").innerHTML;
+    var b4 = document.getElementById("b4").innerHTML;
+    var b5 = document.getElementById("b5").innerHTML;
+    var b6 = document.getElementById("b6").innerHTML;
+    var b7 = document.getElementById("b7").innerHTML;
+    var b8 = document.getElementById("b8").innerHTML;
+    var b9 = document.getElementById("b9").innerHTML;
+    
+
+    if(((b1=="X") || (b1=="O")) && ((b1 == b2) && (b2 == b3))){
```

Comment:
get rid if if-else-if ladder and missing DRY

Reply:
Can I use ternary operator instead ?